js/main.js

```diff
@@ -279,8 +279,9 @@ var MM = (function () {
 	 * @param {number} speed The speed of the show animation.
 	 * @param {Function} callback Called when the animation is done.
 	 * @param {object} [options] Optional settings for the show method.
+	 * @param {Function} errorCallback Called when the module failed to show.
 	 */
-	var showModule = function (module, speed, callback, options) {
+	var showModule = function (module, speed, callback, errorCallback, options) {
```

Comment:
woudln't just "error" be the most common name for such a function (here and in the other occurences too)?

Reply:
not sure error sounds more like an object instead of a function

Reply:
yes, you are probably right :-)